lib/engine/game/g_1848/step/buy_sell_par_shares.rb

```diff
@@ -15,6 +15,12 @@ def process_buy_shares(action)
                        allow_president_change: @game.pres_change_ok?(action.bundle.corporation))
             track_action(action, action.bundle.corporation)
           end
+
+          def can_sell?(entity, bundle)
+            return true if bundle.corporation == @game.boe
```

Comment:
Do BoE shares go back to BoE instead of the market?

Reply:
there's no boe market/ipo distinction. In the phyiscal game the shares go into a pile on top of the BOE placemat.
In this implementation i just move all the shares into the market at setup of game.